modules/tinymce/CHANGELOG.md

```diff
@@ -0,0 +1,2442 @@
+- Fixed dialog labels and text color contrast within notifications/alert banners to satisfy WCAG 4.5:1 contrast ratio for accessibility #TINY-3351
+- Fixed selectbox and colorpicker items not being translated #TINY-3546
+- Fixed toolbar drawer sliding mode to correctly focus the editor when tabbing via keyboard navigation #TINY-3533
+- Fixed positioning of the styleselect menu in iOS while using the mobile theme #TINY-3505
+- Fixed the menubutton `onSetup` callback to be correctly executed when rendering the menu buttons #TINY-3547
+- Fixed `default_link_target` setting to be correctly utilized when creating a link #TINY-3508
+- Fixed colorpicker floating marginally outside its container #TINY-3026
+- Fixed disabled menu items displaying as active when hovered #TINY-3027
+
+### Removed
+- Removed redundant mobile wrapper #TINY-3480
+
+## 5.0.3 - 2019-03-19
+
+### Changed
+- Changed empty nested-menu items within the style formats menu to be disabled or hidden if the value of `style_formats_autohide` is `true` #TINY-3310
+- Changed the entire phrase 'Powered by Tiny' in the status bar to be a link instead of just the word 'Tiny' #TINY-3366
+- Changed `formatselect`, `styleselect` and `align` menus to use the `mceToggleFormat` command internally #TINY-3428
+
+### Fixed
+- Fixed toolbar keyboard navigation to work as expected when `toolbar_drawer` is configured #TINY-3432
+- Fixed text direction buttons to display the correct pressed state in selections that have no explicit `dir` property #TINY-3138
+- Fixed the mobile editor to clean up properly when removed #TINY-3445
+- Fixed quickbar toolbars to add an empty box to the screen when it is set to `false` #TINY-3439
+- Fixed an issue where pressing the **Delete/Backspace** key at the edge of tables was creating incorrect selections #TINY-3371
+- Fixed an issue where dialog collection items (emoticon and special character dialogs) couldn't be selected with touch devices #TINY-3444
+- Fixed a type error introduced in TinyMCE version 5.0.2 when calling `editor.getContent()` with nested bookmarks #TINY-3400
+- Fixed an issue that prevented default icons from being overridden #TINY-3449
+- Fixed an issue where **Home/End** keys wouldn't move the caret correctly before or after `contenteditable=false` inline elements #TINY-2995
+- Fixed styles to be preserved in IE 11 when editing via the `fullpage` plugin #TINY-3464
+- Fixed the `link` plugin context toolbar missing the open link button #TINY-3461
+- Fixed inconsistent dialog component spacing #TINY-3436
+
+## 5.0.2 - 2019-03-05
+
+### Added
+- Added presentation and document presets to `htmlpanel` dialog component #TINY-2694
+- Added missing fixed_toolbar_container setting has been reimplemented in the Silver theme #TINY-2712
+- Added a new toolbar setting `toolbar_drawer` that moves toolbar groups which overflow the editor width into either a `sliding` or `floating` toolbar section #TINY-2874
+
+### Updated
```

Comment:
Just wanted to check if this is a valid heading?

Reply:
It is not... we apparently used the wrong phrase back then as well.